docs/docs/guide/debugging.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: debugging
+title: "Debugging Reanimated"
+sidebar_label: "Debugging Reanimated"
+---
+
+import CompatibilityTable from './debugging_table/table';
+
+Due to Reanimated's unique architecture and usage of a second thread debugging
+can be problematic and some common solutions might not work as expected.
+This article summarizes all the available methods and highlights their caveats.
+
+:::note
+
+Reanimated v1 works with all of the common React Native debugging tools. This
+article focuses on the use of these tools with Reaniamted v2.
+
+:::
+
+### React Native debugging tools
+
+These are the debugging tools we checked for compatibility with React Native
+apps using the Reanimated library:
+
+- [**Chrome debugger**](https://reactnative.dev/docs/debugging#chrome-developer-tools)
+(also known as _React Native Debugger_) is a simple tool that creates a web
+worker inside your preferred browser which executes your app's code. It works
+with all the runtimes available in React Native.
+
+- [**Chrome DevTools**](https://reactnative.dev/docs/hermes#debugging-js-on-hermes-using-google-chromes-devtools)
+(unavailable for JSC) are Chrome's built-in feature that allow for connecting to a
+remote JavaScript runtime. This means that the code you are debugging actually
+runs on the device itself and behaves the same way as it would with DevTools detached.
+
+- [**Flipper**](https://fbflipper.com/) (Hermes debugger) is an app created by
+Facebook that makes it easy to use _Chrome DevTools_ while providing additional tools for
+UI inspection and debugging.
+
+- [**Safari DevTools**](https://reactnative.dev/docs/debugging#safari-developer-tools)
+are Safari's built-in feature and are available for iOS devices running JSC only. They work
+very similarly to Chrome DevTools - they allow you to connect to a remote runtime
+and keep code execution on device.
+
+- [**React DevTools**](https://reactnative.dev/docs/debugging#react-developer-tools)
```

Comment:
The official name seems to be "React Developer Tools" (source: https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi). Can we rename this everywhere to avoid confusion and improve searchability?